src/components/CohortBuilder/Summary/index.js

```diff
@@ -2,11 +2,20 @@ import React from 'react';
 import styled from 'react-emotion';
 import Pie from 'chartkit/components/Pie';
 
-import { demographicPiesMock } from './mock';
+import { demographicPiesMock, topDiagnosesBarMock } from './mock';
 import Card from 'uikit/Card';
 import { CardWrapper } from 'uikit/Card/styles';
 import { Col, Row } from 'react-grid-system';
 
+import theme from 'theme/defaultTheme';
```

Comment:
let's not import `theme` directly, there's a `withTheme` hoc that can be imported from `emotion-theming` which provides a `theme` prop.

Reply:
Done'd.